invenio_accounts/testutils.py

```diff
@@ -66,6 +66,27 @@ def create_test_user(email='test@test.org',
     return user
 
 
+def create_legacy_user(email='test@test.org', password='123456', **kwargs):
```

Comment:
I.e. you can move this method to the pytest helpers since this file is for code that other modules can import.